pylal/coh_PTF_pyutils.py

```diff
@@ -85,7 +85,7 @@ def get_bestnr( trig, q=4.0, n=3.0, null_thresh=(4.25,6), snr_threshold=6.,\
         sens[ifo] = getattr(trig, 'sigmasq_%s' % i.lower()) * \
                         sum(numpy.array([fPlus[ifo], fCross[ifo]])**2)
     ifos.sort(key=lambda ifo: sens[ifo], reverse=True)
-    for i in xrange(0,2):
+    for i in xrange(0,len(ifos)):
```

Comment:
This is a *significant* change to the detection statistic. Is this intended? I'm happy with the rest of the patch but a change like this would need to be discussed and agreed at the GRB call.